3587/scrape_info.rb

```diff
@@ -0,0 +1,42 @@
+require './links_modifier.rb'
+# class that get info about gem from git
+class Scrape
+  attr_reader :watch
+  def initialize(page, used_by_page, name)
+    @page = page
+    @watch = [name]
+    find_info_without_used_by
+    @used_by_page = used_by_page
+    find_used_by
+  end
+
+  def find_forks_stars_watched
```

Comment:
In ruby we don't use prefixes for methods name like `get_ set_` you just name method with understanding which resource you will get if you call it. Rename it pls